src/components/RadioButton.js

```diff
@@ -7,12 +7,14 @@ const RadioButton = React.createClass({
     activeButtonStyle: React.PropTypes.object,
     buttonStyle: React.PropTypes.object,
     checked: React.PropTypes.bool,
+    color: React.PropTypes.string,
```

Comment:
We're using `brandColor` as the prop name elsewhere. Let's do that here too for the sake of consistency.